docs/source/services/timelock_service/installation.rst

```diff
@@ -0,0 +1,58 @@
+.. _timelock-installation:
+
+Timelock Server Installation
+============================
+
+1. Clone the git repository.
+
+   .. code:: bash
+
+      git clone git@github.com:palantir/atlasdb.git; cd atlasdb
+
+2. Generate a tar file with Gradle:
```

Comment:
We should be publishing this somewhere. @jboreiko for more info on how we are going to do this. (We did have it in the codebase but it broke internal publishing, and maybe external publishing)